scss/_mixins.scss

```diff
@@ -22,24 +31,32 @@
 	}
 }
 
-// Base styles for all icons
+/// Base styles for all icons
 @mixin oFtIconsBaseIconStyles() {
 	font-family: $_o-ft-icons-font-name;
 	display: inline-block;
 	vertical-align: middle;
 	// These are marked as !important as the icon font is not designed to be used with faux bold or faux italic styles
 	// If bolder versions of an icon are required a new SVG should be designed and added to the icon font
+	// scss-lint:disable ImportantRule
 	font-weight: normal !important;
 	font-style: normal !important;
 	speak: none;
 	text-decoration: inherit;
 	text-transform: none;
 	text-rendering: optimizeLegibility;
+	// scss-lint:disable VendorPrefix VendorPrefixes
```

Comment:
Note that this is to be forward compatible with SCSS-Lint v0.35.0 where `VendorPrefixes` becomes `VendorPrefix`